src/utils/groupByUniqName.js

```diff
@@ -0,0 +1,17 @@
+import { isEmpty, chain, has } from 'lodash'
+
+export const groupByUniqName = (value, field = '') => {
+  const makeUniq = items => {
+    return chain(items)
+      .filter(item => has(item, field + '.name') || has(item, 'name'))
+      .uniqBy(field + '.name')
```

Comment:
I don't understand this.
How does the `uniqBy(field + '.name')` work for the items that have no `field + '.name'` but only have `'name'`?